src/Notification/Service/NotificationService.php

```diff
@@ -54,8 +89,8 @@ public function listNotifications(FilterParameter $parameters): Collection
         foreach ($listing as $listEntry) {
             $entry = $this->notificationHydrator->hydrate($listEntry);
             $this->eventDispatcher->dispatch(
-                new NotificationEvent($entry),
-                NotificationEvent::EVENT_NAME
+                new NotificationListEvent($entry),
```

Comment:
Why do we need two different events here?

Reply:
Because they work with different elements (NotificationListItem and Notification). NotificationListItem contains only data for the tree